### DISCUSSION TITLE
X/Y Offset

### DISCUSSION THREAD

**Discussion by Toihva** [Jyers/Marlin#72] [Q&A] [answered]
I got this FW up and running and see it as one I want to keep using.  Since I have a BL touch and went with solid bed mounts I ran a  bed visualizer and it was [pretty bad](https://gyazo.com/8efbf9af272f689d23685716f8a3eef0).  

I went and checked to see if the probe was off the build plate, but as I watched, it started off pretty good, starting the probe at roughly where the screw is under the plate.  But at the backend it has under 1mm before it is off the plate.  So wondering if X/Y offset needs to be improved or if the bedsize is off in the FW.

Thanks for any help

**Comment by Jyers** [comment, marked as answer]
The bed size in the firmware is 230 by 230, so unless your endstops are really off that shouldn't be the issue. Most likely your Y offset is a bit off, causing the mesh to be off center in that direction.
